docs/src/main/sphinx/release/release-430.md

```diff
@@ -0,0 +1,47 @@
+# Release 430 (20 Oct 2023)
+
+## General
+
+* Improve performance of freshness calculation for materialized views. ({issue}`19423`)
+* Fix ordering of returned result set when query has a window function with
+  identical ordering. ({issue}`19399`)
+* Fix incorrect results for query involving an aggregation in a correlated
+  subquery. ({issue}`19002`)
+* Fix `RunningQueries` metric to represent all running queries. ({issue}`19427`)
```

Comment:
See PR ... one of the metrics was changed from "Progressing" to "Running" queries  .. I think this might surface via JMX, system metrics and maybe web UI

Reply:
Only JMX .. leaving it out.